tests/InputValidatorTest.php

```
@@ -170,53 +176,61 @@ public function validateCustomCallbacksThrowsForEmptyPath(): void
         InputValidator::validateAuditLogger(null);
-        $this->assertTrue(true);
     }
 
     #[Test]
     public function validateAuditLoggerPassesForCallable(): void
     {
+        // Void validator: passing means not throwing. Declared explicitly rather than
+        // faked with a vacuous assertion; rejection is covered by the matching negative tests.
+        $this->expectNotToPerformAssertions();
+
         InputValidator::validateAuditLogger(fn($field, $old, $new): null => null);
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$field'. (undefined)

(UnusedFormalParameter)